CONTRIBUTING.md

```diff
@@ -17,19 +17,70 @@ Before submitting a new issue, do the following:
 - Verify you have Xcode tools installed by running `xcode-select --install`.
 - Make sure to read through the [README](https://github.com/fastlane/fastlane) of the project.
 
-
 When submitting a new issue, please provide the following information:
```

Comment:
Potentially? Let's deal with that when we decide to merge the issue template PR.